tests/Api/Admin/ProductVariantTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Api\Admin;
+
+use Sylius\Tests\Api\JsonApiTestCase;
+use Exception;
+use Symfony\Component\HttpFoundation\Response;
+
+class ProductVariantTest extends JsonApiTestCase
+{
+    /**
+     * @test
+     *
+     * @throws Exception
```

Comment:
Does it?

Reply:
There is always this 0.01% of chance that it will 👀